src/dlgTriggerEditor.h

```diff
@@ -577,8 +577,9 @@ private slots:
     QAction* mpAction_restoreEditorActionsToolbar = nullptr;
     QAction* mpAction_restoreEditorItemsToolbar = nullptr;
 
-    // We need to keep a record of this button as we have to disable it
+    // We need to keep a record of these buttons as we have to disable them
     // for the "Variables" view:
+    QAction* mpToggleActiveAction = nullptr;
```

Comment:
For consistency (and to follow what the Qt Designer tool tends to call them) we, perhaps ought to be calling this `mpAction_toggleActiveAction` - however since there is more than just this one that'll probably have to be left to a separate "clean-up" PR...